buildutils/src/ensure-package.ts

```diff
@@ -31,22 +31,6 @@ import { LabIcon } from './labicon';
 {{labiconConstructions}}
 `;
 
-const ICON_CSS_CLASSES_TEMPLATE = `
-/**
- * (DEPRECATED) Support for consuming icons as CSS background images
- */
-
-/* Icons urls */
-
-:root {
-  {{iconCSSUrls}}
-}
-
-/* Icon CSS class declarations */
-
-{{iconCSSDeclarations}}
-`;
-
```

Comment:
Could you elaborate why this is needed?
I imagine we could remove this for JLab 4.0. But as it may impact extensions, I would recommend doing it in a separate PR in case we need to revert it (and to have a better git history).

Reply:
The idea was to stop including classes for new icons. Since this is run as a part of the integrity check, I wasn't able to manually exclude that icon classes from being added; excluding this will add the new icon class in `deprecated.css`. However, agree that creating a separate PR will be easier to track in case of issues.